middle_end/flambda/types/env/aliases.ml

```diff
@@ -715,10 +946,11 @@ Format.eprintf "looking for canonical for %a, candidate canonical %a, min order
   Name_mode.print min_name_mode;
 *)
   let find_earliest () =
-    (* There used to be a shortcut that avoided consulting the aliases in the
+    (* There used to be a shortcut that avoided a consulting the aliases in the
        common case that [element] is itself canonical and has no aliases, since
        then it does not appear in [canonical_elements]. However, this shortcut
-       was broken: a canonical element *with* known aliases may still not appear
+       was broken: a canonical element *with* known aliases may still not appea
```

Comment:
`appea\n` -> `appear`. This might be an artifact of the split, sorry about that.